.github/dependabot.yml

```diff
@@ -5,23 +5,23 @@
 
 version: 2
 updates:
-  - package-ecosystem: "pip"
```

Comment:
This `.github` directory had been added to the `.prettierignore` file, so I removed it and formatted these files.